cmd/git-sync/main.go

```diff
@@ -238,8 +239,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	if (*flUsername != "" || *flPassword != "" || *flCookieFile || *flAskPassURL != "") && *flSSH {
-		fmt.Fprintf(os.Stderr, "ERROR: --ssh is set but --username, --password, --askpass-url, or --cookie-file were provided\n")
+	if (*flUsername != "" || *flPassword != "" || *flPasswordFile != "" || *flCookieFile || *flAskPassURL != "") && *flSSH {
+		fmt.Fprintf(os.Stderr, "ERROR: --ssh is set but --username, --password, --password-file, --askpass-url, or --cookie-file were provided\n")
 		os.Exit(1)
 	}
```

Comment:
Probably need a block to ensure that user doesn't set both flPassword and flPasswordFile at the same time

Reply:
maybe also a block to check that one of flPassword and flPasswordFile are set when flUsername is set ?